udts.py

```diff
@@ -210,6 +211,14 @@ def save_as_state():
         save_config(window.config)
 
 
+def edit_points():
+    pts_on_win = int(window.points_on_win_spinbox.text())
+    pts_on_tie = int(window.points_on_tie_spinbox.text())
+    pts_on_loss = int(window.points_on_loss_spinbox.text())
+    broadcast.edit_points([pts_on_win, pts_on_tie, pts_on_loss])
+    force_refresh_ui()
+
+
```

Comment:
should probably consolidate on how these should be stored and accessed. if list then let's use a list everywhere. ideally this should be un-fuck-up-able as possible. separate values? a configuration object that stores them separately? not sure, but probably not a great habit to get into of passing lists like this where position dictates interpretation